burn-autodiff/src/ops/module.rs

```diff
@@ -158,6 +158,28 @@ impl<B: Backend> ModuleOps<ADBackendDecorator<B>> for ADBackendDecorator<B> {
         }
     }
 
+    fn conv_transpose2d(
+        _x: ADTensor<B, 4>,
+        _weight: ADTensor<B, 4>,
+        _bias: Option<ADTensor<B, 1>>,
+        _stride: [usize; 2],
+        _padding: [usize; 2],
+        _out_padding: [usize; 2],
+    ) -> ADTensor<B, 4> {
+        panic!("Transposed 2D convolution doesn't yet support backward.");
```

Comment:
Something I learned recently: using `todo!` macro panics and some IDEs convey as a todo (maybe also useful to search the code)